typescript/examples/vercel-ai/uniswap/.env.template

```diff
@@ -0,0 +1,3 @@
+# Uniswap (get an API key by running the `pnpm create-api-key` command)
```

Comment:
Is this script pushed?

Reply:
Nope, my but - I copied it over from a template I used